src/components/molecules/AdminSpaceCard/AdminSpaceCard.scss

```diff
@@ -53,27 +61,14 @@ $admin-venue-card-button-margin: 5px 0;
   }
 
   &__logo {
-    height: 70px;
-    width: 70px;
-    border-radius: 50%;
+    @include square-size($admin-space-card-logo-size);
```

Comment:
`$admin-space-card-logo-size: 80px;` and it had `height: 70px` previously. Is it right?

Reply:
I changed it a bit. Tried to empirically keep it looking the same even if values are different.